### PR TITLE
Update to unqualified-search-registries enhancement

### PR DESCRIPTION
**Pull request by umohnani8**
Update the enhancement with information regarding other
components that may be affected by this API change.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@bparees @adambkaplan @dmage PTAL

File: enhancements/api-review/add-unqualified-search-registries.md [hunk 1]
```diff
@@ -129,11 +132,14 @@ Note: adding a drop-in file at `/etc/containers/registries.conf.d` completely ov
...
 
-No new risks are introduced. The `unqualified-search-registries` field already exists in the `/etc/containers/registries.conf` file on all nodes in a cluster. This enhancement just adds the ability for the user to be able to configure this list. A tiny api change for adding the `UnqualifiedSearchRegistry` field to the images.config.openshift.io CR is needed.
+Need to ensure that all the components that deal with image management are updated to handle this new option so that they are all on the same page. The `unqualified-search-registries` field already exists in the `/etc/containers/registries.conf` file on all nodes in a cluster. This enhancement just adds the ability for the user to be able to configure this list. An API change for adding the `UnqualifiedSearchRegistry` field to the images.config.openshift.io CR is needed.
+Builds, Imagestream imports, and image-registry-pull-through will have to handle `unqualified-search-registries` as well to ensure that we are consistent across all the components that deal with image pull and management.
 Also, CRI-O will be doing the image search and will have access to the credentials for all the registries in the list.
```
Comment (umohnani8) [inline enhancements/api-review/add-unqualified-search-registries.md:143]: @dmage will imports and registry-pull through have access to the same credentials as cri-o does?

Comment (bparees) [reply enhancements/api-review/add-unqualified-search-registries.md:143]: i know for sure that imagestream import will now utilize the node creds.  i'm less sure about pull-through but i believe it also does.  That was part of what we were finally making consistent in the last couple releases (using the same creds for import, pullthrough, and build image pulls as we do for pod image pulls).

Comment (bparees) [reply enhancements/api-review/add-unqualified-search-registries.md:143]: @adambkaplan may also know the answer here.

**Comment by bparees** [comment]
/approve

this lgtm but will let Adam and Oleg confirm.

File: enhancements/api-review/add-unqualified-search-registries.md (hunk 1 above)
Comment (dmage) [reply enhancements/api-review/add-unqualified-search-registries.md:143]: Yes, pull-through also supports installer secrets, it was implemented by DEVEXP-515

**Pull request merged**